backend/tests/unit/models/test_token_aggregations.py

```
@@ -0,0 +1,392 @@
+"""
+Unit tests for token usage aggregation functionality added in EP-0009.
+
+Tests the computed fields in DetailedStage and SessionStats models that
+aggregate token usage from LLM interactions.
+"""
+
+import pytest
+from unittest.mock import Mock
+
+from tarsy.models.history_models import DetailedStage, SessionStats, ChainStatistics, LLMTimelineEvent
```

Ruff (0.12.2)

Line too long (102 > 88)

(E501)